de.dlr.sc.virsat.model.extension.requirements.test/.checkstyle

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Features usually dont have checkstyle file since they dont containt source code.

Reply:
Agree. But hits is tests. Tests should be checkstyle tested.